ui/vty/app/Glue.hs

```diff
@@ -102,16 +102,17 @@ knitFaceToUI UiFace{..} KnitFace{..} =
       UiBalance ->
         Right $ Knit.ExprProcCall
           (Knit.ProcCall Knit.balanceCommandName [])
-      UiSend address amount passphrase -> do
-        argAddress <- decodeTextAddress address
-        argCoin <- readEither amount
+      UiSend inputs passphrase -> do
```

Comment:
Parameter says `inputs`, but you are constructing `tx-out`. Huh?

Reply:
It's nice when someone actually reads your code